html/host.html

```diff
@@ -3,66 +3,100 @@
-          <option>[1] Bug Hunters Competition</option>
-          <option>[1] Dice Stalagmite</option>
-          <option>[1] Fish Spotters</option>
-          <option>[1] Gridlock</option>
-          <option>[1] Guppy Spots</option>
-          <option>[1] PANDA BEAR</option>
-          <option>[1] Polling</option>
-          <option>[1] Polling Advanced</option>
-          <option>[1] Prisoners Dilemma</option>
-
-          <option>[a] Oil Cartel</option> <!-- requires narrowcast plots -->
-
-          <option>[0] Restaurants</option> <!-- makes no distinction between AI restaurants and player-owned ones -->
-
-          <option>[0] Bird Breeders</option> <!-- requires spectator clients -->
-          <option>[0] Gridlock Alternate</option> <!-- requires spectator clients -->
-
-          <option>[0] Client Overrides Example</option> <!-- requires view overrides -->
-          <option>[0] Critter Designers</option> <!-- requires view overrides -->
-          <option>[0] Current in a Wire</option> <!-- requires view overrides -->
-          <option>[0] MTG 1 Equal Opportunities</option> <!-- requires view overrides -->
-          <option>[0] MTG 2 Random Assignment</option> <!-- requires view overrides -->
-          <option>[0] MTG 3 Feedback Loop</option> <!-- requires view overrides -->
-          <option>[0] Sampler</option> <!-- requires view overrides -->
-        </select>
-        <label>
-          Session Name:
-          <input name="sessionName" type="text" pattern="\w.*" required>
-        </label>
-        <label>
-          Password (optional):
-          <input name="password" type="password">
-        </label>
-        <input type="submit" value="Launch">
+
+        <!-- Select activity div -->
+        <div class="activity-selection">
+
+          <!-- Activity selection + description  -->
+          <div class="activity-description">
+            <div class="library-model-container" class="library-model-container">
+              <select class="library-model" id="library-model" name="libraryModel" type="chooser"></select>
+            </div>
+            <p id="activity-text-short" class="activity-text-short"></p>
+            <button id="read-more-btn" class="read-more-btn" type="button">Read More</button>
+          </div>
+
+          <!-- Activity image -->
+          <div class="img-wrapper">
+            <img id="activity-img" class="img" alt="Activity Image">
+          </div>
+        </div>
+
+        <!-- Other form inputs (session name, password) -->
+        <div class="form-input-wrapper">
+          <div>
+            <label class="form-label" for="sessionName">Session Name</label>
+            <input class="form-input" name="sessionName" type="text" pattern="\w.*" required>
+          </div>
+          <div class="center-input">
+            <label class="form-label" for="password">Password</label>
+            <input class="form-input" name="password" type="text" required>
```

Comment:
My preferred style is:

```html
<label>Password
  <input>
</label>
```

instead of:

```html
<label for="password">Password</label>
<input name="password">
```

Reply:
Not sure how much this will bother you, but if I change it, then it messes up some of the styling I have with the `<label>` and `<input>` (I added a custom amount of padding, which is gone if I "condense" the `<input>` into the `<label>`).